### DISCUSSION TITLE
Is there any way to deploy Production when a commit is still in PR ?

### DISCUSSION THREAD

**Discussion by kentnathaniel** [BetaHuhn/deploy-to-vercel-action#153] [Q&A] [answered]
I have a case where I want to deploy do Production deployment even though a commit is still in PR. 
But it seems like all the commits in PR are always targeting Preview environment.

Am I missing something because I can't find any related example about this in the documentation?

**Comment by BetaHuhn** [comment, marked as answer]
Hi @kentnathaniel,

there is a `PRODUCTION` input which tells the Vercel CLI if a deployment should be targeting the production environment. At the moment this gets overwritten to `false` if the action is run inside a PR. This is the intended behavior, but I can see why this might be unexpected and I think it would be better of the `PRODUCTION` input also affects the deployment environment in a PR.

I will work on a fix.

**Reply by BetaHuhn** [reply]
Done, fixed in [v1.8.0](https://github.com/BetaHuhn/deploy-to-vercel-action/releases/tag/v1.8.0).

You can use the `PRODUCTION` input to tell the action what environment to deploy to. It will default to false for PRs, but now the value provided by the user will take precedence.

**Reply by kentnathaniel** [reply]
nice ! thanks a lot @BetaHuhn

**Reply by BetaHuhn** [reply]
No problem!
